### ISSUE TITLE
Update Map's Estimated Time Style

### ISSUE THREAD

**Issue by virtual** [freeCodeCamp/freeCodeCamp#16321]
#### Challenge Name
Any challenge showing the [map](https://beta.freecodecamp.org/en/challenges/applied-accessibility/use-headings-to-show-hierarchical-relationships-of-content)

#### Issue Description
The estimated time is not accessible with the current color `#bbbbbb`; since it is also rather large and taking up some real estate, I'd propose making it darker `#555555` to pass WCAG 2.0 AA. This will also allow it to be smaller and normal font style as it's not so light and hard to read at a small font size.

#### Proposed Style Code
```css
.map-block-time {
    color: #555;
    font-weight: normal;
    font-size: 1.4rem;
}

```
#### Screenshot
Proposed update:
![Estimated challenge time proposed design](https://i.imgur.com/V5yhDFP.png)

**Comment by vkWeb** [comment]
@virtual I think the font-size is okay and the color thing should be implemented according to WCAG 2.0 AA principles. `#555555` looks good 😄.
If @QuincyLarson, @Bouncey, @BerkeleyTrue  agrees with the change. I'll raise a PR for this.

**Comment by QuincyLarson** [comment]
@vkWeb Yes - I think that color works. Do you know much about color theory? We have different recommended shades on https://design-style-guide.freecodecamp.org/ but it looks like we may need to add a new shade of darker gray. Do you know which shade would be dark enough and would also jive with our other colors and shades there?

**Comment by virtual** [comment]
@QuincyLarson Thanks for the reference to your guide! It is a bit ironic but `#555555` is actually also the color of the left-side navigation used in your style guide. :)

**Comment by vkWeb** [comment]
@QuincyLarson I don't have that much knowledge about color theory. But as far as accessibility concern I think `#555555` will look better. We should add `#555555` in the secondary color list of our design style guide. 

@virtual Yes. That's true.

**Comment by QuincyLarson** [comment]
@vkWeb OK - great. Sounds like a plan. Can you make a pull request to update the style guide as well? https://github.com/freecodecamp/design-style-guide

**Comment by vkWeb** [comment]
@QuincyLarson I'm very excited. It'll be my first contribution to open source code base.  I'll raise a PR within 2 days. 😃

**Comment by vkWeb** [comment]
@QuincyLarson  I have raised a [Pull Request](https://github.com/freeCodeCamp/design-style-guide/pull/17) in the [design-style-guide](https://github.com/freeCodeCamp/design-style-guide) repository. I'll appreciate your feedback.
